README.md

```diff
@@ -344,63 +337,98 @@ ctkey --username=$CTKEY_USERNAME \\
 --iam-role=$CT_AWS_ROLE setenv
 ```
 
-Eventually, we will move this over to wrapper
-so developers do not need to manually run these commands.
+Eventually, we will move this over to wrapper so developers do not need to
+manually run these commands.
 
-### Live Reload Go with Air (Optional)
+## Development and Debugging
 
-If you want to reload the Go application on changes locally,
-you can use [Air](github.com/cosmtrek/air)
+### Authentication
 
-Install it:
+The application has two authentication modes. The main mode is to use Okta to
+authenticate using hosted services. The second is to use a local-only login mode
+that avoids this network dependency.
 
-```bash
-go get -u github.com/cosmtrek/air
-```
+To sign in using local mode, Click the **Use Local Auth** button on the sign in
+page. This is only provided when running the app locally.
 
-Run it:
+To enable Okta authentication locally, add the following values to
+`.envrc.local`:
 
 ```bash
-air
+export OKTA_TEST_USERNAME=
+export OKTA_TEST_PASSWORD=
+export OKTA_TEST_SECRET=
 ```
 
-It's not currently set up to run with docker.
-You can edit the config [here](.air.conf)
+These values can be found in 1Password.
 
-### Setup: Minio
+### GraphQL Playground
 
-[MinIO](https://min.io/) is an S3 compatible object store.
-It ships as a Docker container and accepts normal AWS S3
-API requests. This allows us to test file uploading functionality
-in our local development environments without needing to interact
-with CMS AWS accounts.
+You can visit `http://localhost:8080/api/graph/playground` to access a GraphQL
+playground while the Go backend is running. **You will need to enter
+`/api/graph/query` as the query path in the UI for this to work.**
 
-The container is configured as part of our `docker-compose.yml` and
-should be running when you `docker-compose up`. You'll need to set
-two environment variables locally in your `.envrc.local` file:
+### Accessing the application over Tailscale
```

Comment:
I'm wondering if a 1-liner would be helpful here explaining what Tailscale is, or why on  would want to access the app over tailscale.  I don't know the answer, though maybe it's common developer knowledge that I've just not encountered.